_data-prepper/pipelines/configuration/processors/mutate-event.md

```diff
@@ -0,0 +1,304 @@
+---
+layout: default
+title: Mutate event
+parent: Processors
+grand_parent: Pipelines
+nav_order: 45
+---
+
+# Mutate event processors
+
+Mutate event processors allow you to modify events in Data Prepper. The following processors are available:
+
+* [AddEntries](#addentries): Allows you to add entries to an event.
+* [CopyValues](#copyvalues): Allows you to copy values within an event.
+* [DeleteEntry](#deleteentry): Allows you to delete entries from an event.
+* [RenameKey](#renamekey): Allows you to rename keys in an event.
+* [ConvertEntry](#convertentry): Allows you to convert value types in an event.
+
+## AddEntries
+
+The `AddEntries` processor adds entries to an event.
+
+### Configuration
+
+You can configure the `AddEntries` processor with the following options.
+
+| Option | Required | Description |
+| :--- | :--- | :--- |
+| `entries` | Yes | A list of entries to add to an event. |
+| `key` | Yes | The key of the new entry to be added. Some examples of keys include: `my_key`, `myKey`, and `object/sub_Key`. |
+| `value` | Yes | The value of the new entry to be added. You can use the following data types: strings, booleans, numbers, null, nested objects, and arrays. |
+| `overwrite_if_key_exists` | No | When set to `true`, the existing value is overwritten if `key` already exists in the event. The default value is `false`. |
+
+### Usage
+
+To get started, create the following `pipeline.yaml` file:
+
+```yaml
+pipeline:
+  source:
+    file:
+      path: "/full/path/to/logs_json.log"
+      record_type: "event"
+      format: "json"
+  processor:
+    - add_entries:
+        entries:
+        - key: "newMessage"
+          value: 3
+          overwrite_if_key_exists: true
+  sink:
+    - stdout:
+```
+{% include copy.html %}
+
+
+Next, create a log file named `logs_json.log` and replace the `path` in the file source of your `pipeline.yaml` with this filepath. For more information, see [Configuring Data Prepper]({{site.url}}{{site.baseurl}}/data-prepper/getting-started/#2-configuring-data-prepper).
```

Comment:
In the first sentence, if we mean to replace `path` with `logs_json.log`, then it should be "with that filepath."